src/Field/WfsSearch/WfsSearch.spec.jsx

```diff
@@ -0,0 +1,169 @@
+      wrapper.instance().doSearch.mockReset();
+    });
+  });
+
+  describe('#onFetchSuccess', () => {
+    it('sets the response features as state.data', () => {
+      const wrapper = TestUtil.mountComponent(WfsSearch);
+      const features = [{
+        id: '752526',
+        properties: {
+          name: 'Deutschland'
+        }
+      }];
+      wrapper.instance().onFetchSuccess({features});
+      expect(wrapper.state().data).toEqual(features);
+    });
+  });
+
+  describe('#onFetchError', () => {
+    it('sets the response as state.data', () => {
+      const wrapper = TestUtil.mountComponent(WfsSearch);
+      const loggerSpy = jest.spyOn(Logger, 'error');
+      wrapper.instance().onFetchError('Peter');
+      expect(loggerSpy).toHaveBeenCalled();
+      expect(loggerSpy).toHaveBeenCalledWith('Error while requesting WFS GetFeature: Peter');
+      loggerSpy.mockReset();
+      loggerSpy.mockRestore();
+    });
+  });
+
+  describe('#onMenuItemSelected', () => {
+    it('calls this.props.onSelect with the selected item', () => {
+      //SETUP
+      const data = [{
+        id: '752526',
+        properties: {
+          name: 'Deutschland'
+        }
+      }];
+      const map = new OlMap({
+        layers: [new OlLayerTile({name: 'OSM', source: new OlSourceOsm()})],
+        view: new OlView({
+          projection: 'EPSG:4326',
+          center: [37.40570, 8.81566],
+          zoom: 4
+        })
+      });
+      //SETUP END
+
+      const selectSpy = jest.fn();
+      const wrapper = TestUtil.mountComponent(WfsSearch, {
+        onSelect: selectSpy,
+        map
+      });
+      wrapper.setState({
+        data: data
+      });
+      wrapper.instance().onMenuItemSelected('752526');
+      expect(selectSpy).toHaveBeenCalled();
+      expect(selectSpy).toHaveBeenCalledWith(data[0], map);
+
+      selectSpy.mockReset();
+      selectSpy.mockRestore();
+    });
+  });
+
+  describe('default #onSelect', () => {
+    it('zooms to the selected feature', () => {
+      jest.useFakeTimers();
+      //SETUP
+      const feature = {
+        type: 'Feature',
+        id: '752526',
+        properties: {
+          name: 'Peter',
+        },
+        geometry: {
+          type: 'Polygon',
+          coordinates: [[[10, 40],[40, 40],[40, 10],[10, 10],[10, 40]]]
+        }
+      };
+      const map = new OlMap({
+        layers: [new OlLayerTile({name: 'OSM', source: new OlSourceOsm()})],
+        view: new OlView({
+          projection: 'EPSG:4326',
+          center: [37.40570, 8.81566],
+          zoom: 4
+        })
+      });
+      //SETUP END
+
+      const wrapper = TestUtil.mountComponent(WfsSearch, {map});
+      const fitSpy = jest.spyOn(map.getView(), 'fit');
+      wrapper.props().onSelect(feature, map);
+      expect(fitSpy).toHaveBeenCalled();
+      setTimeout(() => {
+        expect(map.getView().getCenter()).toEqual([25, 25]);
+        expect(map.getView().getZoom()).toEqual(2);
```

Comment:
is this actually tested?

Reply:
`expect.assertions` is missing. I'll add it.